adsmp/tasks.py

```diff
@@ -191,6 +191,11 @@ def reindex_records(bibcodes, force=False, update_solr=True, update_metrics=True
             logger.error('The bibcode %s doesn\'t exist!', bibcode)
             continue
 
+        # Adding boost factors to r to pass it to solr
```

Comment:
I'm not sure whether this works. This was the only way I could think of where we can access the contents of BoostFactors DB and pass it to solr_updater. But this does not seem like good practice.